src/fs/copy.js

```diff
@@ -1,5 +1,31 @@
+import { existsSync, mkdirSync, readdirSync, copyFileSync } from 'fs';
+import { resolve } from 'path';
+
 const copy = async () => {
-    // Write your code here 
+  const sourceFolderPath = resolve('src', 'fs', 'files');
```

Comment:
It's easier to write it as src/fs/files. But it's even better to use `new URL('files', import.meta.url)` or any other way of using script-related files.

Reply:
Fistly, I used the way you point on `src/fs/files`, but while it might be suitable for simple cases, however, it's a relative path and might not work consistently across different environments (Windows, Unix) or if the structure of project changes. It's less explicit

Сoncerning using new URL('files', import.meta.url) it might be useful, especially if you need a URL, but it might be it seems like it less commonly used for file paths in Node.js applications compared to `path.resolve()`